examples/pytorch/rgcn/experimental/entity_classify_dist.py

```diff
@@ -453,14 +463,14 @@ def run(args, device, data):
             if args.sparse_embedding and not args.dgl_sparse:
```

Comment:
dgl_sparse also need to call zero_grad()

Reply:
dgl's sparse optimizer doesn't have zero_grad. what does pytorch's zero_grad do?

Reply:
https://github.com/dmlc/dgl/blob/062f59a8f6e2b7c96aedab0e5810ae75364492cf/python/dgl/optim/pytorch/sparse_optim.py#L226

zero_grad is to clear the grad record. Otherwise, the grads are accumulated across multi iterations.

Reply:
i added `zero_grad` in the distributed optimizer.